### PR TITLE
Add crew constants from environment variables

### PR DESCRIPTION
**Pull request by uberhacker**
Something that was bothering me.

### CODE REVIEW

**Comment by supechicken** [comment]
Will this break some package setting `CREW_NOT_STRIP` or `CREW_CONFLICTS_ONLY_ADVISORY`like `chrome`? (those env variable are fixed on start, not check every time)

File: lib/const.rb [hunk 1]
```diff
@@ -48,25 +52,44 @@
...
+
+CREW_SHRINK_ARCHIVE = ENV['CREW_SHRINK_ARCHIVE']
+
+# Set testing constants from environment variables
+crew_testing_repo = ENV['CREW_TESTING_REPO']
+crew_testing_branch = ENV['CREW_TESTING_BRANCH']
+crew_testing = ENV['CREW_TESTING']
+crew_testing = '0' if crew_testing_repo.nil? || crew_testing_repo.empty?
+crew_testing = '0' if crew_testing_branch.nil? || crew_testing_branch.empty?
+CREW_TESTING = crew_testing
```
Comment (supechicken) [inline lib/const.rb:80]: Does we need both variables?

**Comment by satmandu** [comment]
Would it be easier to just import all CREW_ env variables as CREW constants?

Or import from a whitelist? That way we don't need to update const.rb?

**Comment by satmandu** [comment]
"Easier" 

i.e. If a CREW_* env variable exists, set it as a ruby constant, and override any other setting of the constant in const.rb, unless in a blacklist of constants. (could go whitelist or blacklist.)

File: lib/const.rb (hunk 1 above)
Comment (satmandu) [reply lib/const.rb:80]: let's merge this and then we can back out a duplicate if necessary....

**Comment by satmandu** [comment]
> Will this break some package setting `CREW_NOT_STRIP` or `CREW_FHS_NONCOMPLIANCE_ONLY_ADVISORY` like `chrome`? (those env variable are fixed on start now, not checked by every time)

The variable names are not changing, so I think it should be ok?

**Pull request merged**